js/constants/constants.js

```diff
@@ -249,7 +254,18 @@ module.exports = {
   UI: {
     defaultScrubberBarHeight: 4,
     DEFAULT_SCRUBBERBAR_LEFT_RIGHT_PADDING: 15,
-    MAX_BUFFERING_SPINNER_DELAY: 60000 // Max allowed value of bufferingSpinnerDelay in milliseconds
+    MAX_BUFFERING_SPINNER_DELAY: 60000, // Max allowed value of bufferingSpinnerDelay in milliseconds
+    DEFAULT_SKIP_BACKWARD_TIME: 10, // In seconds
+    DEFAULT_SKIP_FORWARD_TIME: 10, // In seconds
+    MIN_SKIP_TIME: 1, // Min allowed value for skip controls, in seconds
+    MAX_SKIP_TIME: 60, // Max allowed value for skip controls, in seconds
+    // If the previous video button is clicked more than once, this
+    // is the maximum amount of time (in milliseconds) between calls that would
+    // cause the controller to request the previous video instead of rewinding
+    REQUEST_PREVIOUS_TIME_TRESHOLD: 1500,
+    // When the previous video button is clicked, if the playhead value (in seconds) is
+    // below this constant the controller will request the previous video instead of rewinding
+    REQUEST_PREVIOUS_PLAYHEAD_TRESHOLD: 2
```

Comment:
@jpfranco misspelled "threshold" in the constant name

Reply:
Fixed. Good catch, I think I've never spelled this word properly